test/tests/pbs_smoketest.py

```diff
@@ -1414,36 +1415,33 @@ def test_fairshare_enhanced(self):
         # Check the scheduler usage file whether it's updating or not
         self.server.manager(MGR_CMD_SET, SERVER, {'scheduling': 'False'})
         J1 = Job(TEST_USER4, attrs=job_attr)
-        J1.set_sleep_time(4)
         J2 = Job(TEST_USER2, attrs=job_attr)
-        J2.set_sleep_time(4)
         J3 = Job(TEST_USER1, attrs=job_attr)
-        J3.set_sleep_time(4)
         j1id = self.server.submit(J1)
         j2id = self.server.submit(J2)
         j3id = self.server.submit(J3)
         self.server.manager(MGR_CMD_SET, SERVER, {'scheduling': 'True'})
-        rv = self.server.expect(SERVER, {'server_state': 'Scheduling'}, op=NE)
 
         self.logger.info("Checking the job state of " + j1id)
         self.server.expect(JOB, {'job_state': 'R'}, id=j1id)
         self.server.expect(JOB, {'job_state': 'Q'}, id=j2id)
         self.server.expect(JOB, {'job_state': 'Q'}, id=j3id)
         self.server.manager(MGR_CMD_SET, SERVER, {'scheduling': 'True'})
 
+        self.server.delete(j1id)
         msg = "Checking the job state of " + j3id + ", runs after "
-        msg += j1id + " completes"
+        msg += j1id + " is deleted"
         self.logger.info(msg)
         self.server.expect(JOB, {'job_state': 'R'}, id=j3id)
         self.server.expect(JOB, {'job_state': 'Q'}, id=j2id)
         self.server.manager(MGR_CMD_SET, SERVER, {'scheduling': 'True'})
 
+        self.server.delete(j3id)
         msg = "Checking the job state of " + j2id + ", runs after "
-        msg += j1id + " completes"
+        msg += j1id + " is deleted"
```

Comment:
shouldn't this comment use j3id? because that is what we deleted.

Reply:
Looks to be a copy paste error when the test was initially written.  It's funny since this is a smoke test and one of the tests that is run most often.  We always missed it :)